vllm/model_executor/models/llama.py

```diff
@@ -664,6 +662,7 @@ def compute_logits(
         logits = self.logits_processor(self.lm_head, hidden_states)
         return logits
 
+    # @support_quantized_model_reload_from_hp_weights
     def load_weights(self, weights: Iterable[tuple[str, torch.Tensor]]) -> set[str]:
         loader = AutoWeightsLoader(
             self,
```

Comment:
I assume you will uncomment this decorator, but also why can you not just plug into `AutoWeightsLoader` in order to support all the models that rely on it?